src/app/js/admin/parsing/ParsingExcerpt.js

```diff
@@ -37,13 +39,27 @@ export const getRowStyle = (index, total) => {
     return { opacity, height: 36 };
 };
 
+export const isColumnEnrichment = (columns, enrichments) => {
+    let arrayIsColumnEnrichment = [];
+    columns.map(c => {
+        arrayIsColumnEnrichment[c] = enrichments.find(e => e.name === c);
+    });
+
+    return arrayIsColumnEnrichment;
+};
+
 export const ParsingExcerptComponent = ({
     columns,
     handleAddColumn,
     lines,
     showAddColumns,
+    enrichments,
 }) => {
     const total = lines.length;
+    const arrayIsColumnEnrichment = isColumnEnrichment(
```

Comment:
You could use `useMemo` here to avoid recomputing the array each time we render, but only when columns or enrichments changed. (https://fr.reactjs.org/docs/hooks-reference.html#usememo)

Reply:
That said, I'm not sure it's worth the (little) complexity it adds to create this `isColumnEnrichment`, it may be clearer, to "just" map the enrichment to have a list of names, and use the `includes` function to check if a column is an enrichment.